src/database/seeders/20201118193101-User.js

```diff
@@ -1,24 +1,24 @@
-const {hashPassword} = require("../../utils/hash");
+const {hashPassword}=require("../../utils/hash");
 
 module.exports = {
   up: async queryInterface=> queryInterface.bulkInsert(
     "Users",
     [
       {
-        fullname: "user one",
-        email: "user1@example.com",
-        username:"user1",
+        fullname: "tytyne dusa",
+        email:"dusaflora2@gmail.com",
+        username:"tytyne",
+        isVerified:false,
```

Comment:
```suggestion
        isVerified:true,
```
Because they are asumed as already existing users so they are also verified since they are already signed up!